catalog/PartitionSchemeHeader.hpp

```diff
@@ -180,16 +190,17 @@ class HashPartitionSchemeHeader : public PartitionSchemeHeader {
    * @brief Calulate the partition id into which the attribute value
    *        should be inserted.
    *
-   * @param value_of_attribute The attribute value for which the
-   *                           partition id is to be determined.
+   * @param value_of_attributes The attribute value for which the
+   *                            partition id is to be determined.
    * @return The partition id of the partition for the attribute value.
    **/
   partition_id getPartitionId(
-      const TypedValue &value_of_attribute) const override {
+      const PartitionValues &value_of_attributes) const override {
```

Comment:
In the type definition of ``PartitionValues``, you said it is for representing the boundary for partition values. That doesn't seem true here. Why not just use the vector of ``TypedValue`` instead?